core/dataset_classifier.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+from core.db_client import DBClient
+
+
+class DatasetClassifier(object):
+	
+	def __init__(self):
+		"""
+		Initalizes instance of Dataset Classifier. Only 
```

Comment:
Is this doc unfinished?

Reply:
No, accidentally typed `Only` there. Deleted.